turbo/stages/headerdownload/preverified_hashes_bsc_mainnet.go

```diff
@@ -0,0 +1,8 @@
+package headerdownload
+
+var bscMainnetPreverifiedHashes = []string{
```

Comment:
We should add those only once we have successfully synced to the BSC main net, we have a utility to generate those. Please remove for now